### ISSUE TITLE
annoyance: feeding - changing between bottles and solid 

### ISSUE THREAD

**Issue by Dinth** [babybuddy/babybuddy#804]
Hey. This is not really a bug, but an usability annoyance. My baby is eating solid food while still drinking formula. Changing between formula and solid food (and at the same time between bottle and parent feed) was a hassle which ive been often forgetting about. 
With the new version normal feeding and bottle feeding were separated and i was hoping that the new UI will remove that annoyance, but unfortunately it has not. Clicking "Add feeding" still defaults to bottle feeding if previous feeding was added via "Add bottle feeding" button. 
Could feeding and milk/formula feeding be completely separated from each other now when we have different buttons for each?

**Comment by cdubz** [comment]
Hmmm I'm trying to think how we could improve this... I'm hesitant to remove anything from the feeding entry options because it essentially covers all the available methods. 

So what we have right now is:

## Current state

### Add a Feeding

![image](https://github.com/babybuddy/babybuddy/assets/10456740/68c74dcc-d4fe-4e81-854e-51220dcfc323)

### Add a Bottle Feeding

![image](https://github.com/babybuddy/babybuddy/assets/10456740/bd37d140-6174-4ead-a0b1-422e06c2db10)

@Dinth to be clear -- the U/X you don't like is that the when you select "Add a Feeding" it presets the form values with the last used feeding method values even if you used the "Add a Bottle Feeding" path previously. So primarily the ask here is that the forms independently set the default values?

That is definitely reasonable but yeah I see how a solution would more involve _removing_ things from the "Add a Feeding" form to make this U/X more consistent.

I'm wondering about actually taking this a step further and doing _three_ forms for feedings...

## Three forms state

### Add a Breastfeeding

![image](https://github.com/babybuddy/babybuddy/assets/10456740/1f72f891-a2ff-4269-809a-1b2163a9cee9)

### Add a Bottle Feeding

![image](https://github.com/babybuddy/babybuddy/assets/10456740/d96aefa1-ae08-4183-a872-e42938b2d0df)

### Add an Other Feeding

![image](https://github.com/babybuddy/babybuddy/assets/10456740/9cf36a1e-072d-40ef-bb0b-04967283c2b9)

_I dislike the "Other Feeding" term for the third option so maybe we can come up with something better there._

With these three forms the separate preset options based on the last $type feeding would make a lot more sense.

And this may open up the possibility of adding more options to the "Other" form as has been requested in the past (and I have put off to keep the main form from becoming even more cluttered).

Open to other thoughts/ideas/feedback here.

**Comment by Dinth** [comment]
Hi, thanks for the response. 
Firstly, it would be nice if each screen had its own defaults (previous) value independently. 
Secondly, if im not sure what were the feeding options proposed before, but i think that Breastfeeding, Bottle feeding (formula, breast milk, fortified breast or cows milk) and Solid food would cover everything. There's not much point for another "Other" button which would just list options also available under the other buttons

**Comment by cdubz** [comment]
Yeah with the “other” I was mostly just trying to have a place for the self- or parent-fed *methods* but now that I think about it those could apply to bottle as well. So after adding those to bottle maybe you’re right and the last one could just be “solid”.
